controls/cf_execd.cf

```diff
@@ -16,6 +16,8 @@ body executor control
 
       agent_expireafter  => "$(def.control_executor_agent_expireafter)";
 
+      "runagent_socket_allow_users" => { "$(cfe_internal_permissions._cf_execd_socket_allow_users)" };
```

Comment:
The body attribute name cannot be quoted.

Reply:
oooops!

Reply:
Also, i shouldn't iterate on that list, I should @() it.